app/components/documents/document-card.js

```diff
@@ -74,6 +76,17 @@ export default class DocumentsDocumentCardComponent extends Component {
       && this.currentSession.may('manage-signatures');
   }
 
+  get mayShowEditDropdown() {
+    return (
+      this.args.isEditable
+      && this.currentSession.may('manage-documents')
+      && this.loadSignatureRelatedData.isIdle
+      && this.loadSignatureRelatedData.performCount > 0
+      && (!this.hasSignFlow
+          || (this.hasMarkedSignFlow && !!this.args.decisionActivity))
```

Comment:
This condition is to stop us from showing the dropdown in the `subcase/documents` route. Since we don't have a decisionActivity there, we can't do `Nieuwe versie opladen`, because we can't re-create the sign flow.

We might instead want to only show the options that are possible to execute (`Wijzigen` and `Document verwijderen`).

Reply:
Yes, that's probably a good idea, since the users were not happy about "losing" their features.

Reply:
I would wait for an explicit ask from the users.
I've heard numerous times that "the agenda view" is where secretary will be marking the documents.
If the marking has been toggled than that would have happened in the agenda views.
It makes more sense that adding new versions of documents with markings would also be done in the agenda views.
No dropdown/actions on subcase views docs when a markingActivity is present for now seems ok.